app/src/main/java/com/kylecorry/trail_sense/settings/NavigationSettingsFragment.kt

```diff
@@ -69,6 +72,17 @@ class NavigationSettingsFragment : CustomPreferenceFragment() {
             true
         }
 
+        prefLockScreenPresense?.setOnPreferenceClickListener {
+            if (!prefs.navigation.lockScreenPresence) {
```

Comment:
If my other comment is implemented above, then this should no longer be needed

Reply:
Fixed in https://github.com/kylecorry31/Trail-Sense/pull/677/commits/67b9d5c8d70a29c7902724371c58012f6c2802e8